tornjak-frontend/src/components/dashboard/entries-dashboard-table.js

```diff
@@ -27,65 +27,49 @@ class EntriesDashBoardTable extends React.Component {
     this.SpiffeHelper = new SpiffeHelper();
   }
 
-  workloadEntry(entry) {
-    var thisSpiffeId = this.SpiffeHelper.getEntrySpiffeid(entry)
-    var thisParentId = this.SpiffeHelper.getEntryParentid(entry)
-    // get tornjak metadata
-    var metadata_entry = this.SpiffeHelper.getAgentMetadata(thisParentId, this.props.globalAgents.globalAgentsWorkLoadAttestorInfo);
-    var plugin = "None"
-    var cluster = "None"
-    if (metadata_entry["plugin"].length !== 0) {
-      plugin = metadata_entry["plugin"]
-    }
-    if (metadata_entry["cluster"].length !== 0) {
-      cluster = metadata_entry["cluster"]
-    }
-    // get spire data
-    var admin = this.SpiffeHelper.getEntryAdminFlag(entry)
-    var expTime = "No Expiry Time"
-    if (typeof entry.expires_at !== 'undefined') {
-      var d = new Date(this.SpiffeHelper.getEntryExpiryMillisecondsFromEpoch(entry))
-      expTime = d.toLocaleDateString("en-US", {month: 'short', day: 'numeric', year: 'numeric', hour: 'numeric', minute: 'numeric', second: 'numeric', hour12: false})
-    }
-    return {
-      id: entry.id,
-      spiffeid: thisSpiffeId,
-      parentId: thisParentId,
-      adminFlag: admin,
-      entryExpireTime: expTime,
-      platformType: plugin,
-      clusterName: cluster,
+  entryList() {
+    var filteredData = [], selectedDataKey = this.props.selectedDataKey;
```

Comment:
let's rename this to something like `filterByValue`. From the outside it is selected data, but from the perspective of the table, all it is doing is saying that i'm given something that i want to filter my contents by.